tests/agent_run_spec.rb

```diff
@@ -0,0 +1,43 @@
+describe "Task 8:", host: :agent do
```

Comment:
These tasks should start at 1 for each quest--the task numbering in the content is currently incorrect. Apologies for the mistake there!